azure/Guidelines.md

```diff
@@ -846,110 +847,259 @@ https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#perfo
+return the status monitor for an operation for a subsequent GET on the URL that initiates the LRO, and use this endpoint as
+the status monitor URL returned in the `operation-location` response header.
+
+#### The Status Monitor Resource
+
+All patterns that initiate a LRO either implicitly or explicitly create a [Status Monitor resource](https://datatracker.ietf.org/doc/html/rfc7231#section-6.3.3) in the service's `operations` collection.
+
+<a href="#lro-status-monitor-structure" name="lro-status-monitor-structure">:white_check_mark:</a> **DO** return a status monitor in the response body that conforms with the following structure:
 
 Property | Type        | Required | Description
 -------- | ----------- | :------: | -----------
 `id`     | string      | true     | The unique id of the operation
-`status` | string      | true     | enum that includes values "NotStarted", "Running", "Succeeded", "Failed", and "Canceled"
-`error`  | ErrorDetail |          | Error object that describes the error when status is "Failed"
-`result` | object      |          | Only for POST action-type LRO, the results of the operation when completed successfully
-additional<br/>properties | |     | Additional named or dynamic properties of the operation
+`kind`   | string enum | true(*)  | The kind of operation
+`status` | string enum | true     | The operation's current status: "NotStarted", "Running", "Succeeded", "Failed", and "Canceled"
+`error`  | ErrorDetail |          | If `status`=="Failed", contains reason for failure
+`result` | object      |          | If `status`=="Succeeded" && Action LRO (POST or PUT), contains success result if needed
+additional<br/>properties | | | Additional named or dynamic properties of the operation
 
-<a href="#lro-status-monitor-includes-all-fields" name="lro-status-monitor-includes-all-fields">:white_check_mark:</a> **DO** include the `id` of the operation and any other values needed for the client to form a GET request to the status monitor (e.g. a `location` path parameter).
+(*): When a status monitor endpoint supports multiple operations with different result structures or additional properties,
+the status monitor **must** be polymorphic -- it **must** contain a `kind` property that indicates the kind of long-running operation.
+
+#### Obtaining status and results of long-running operations
+
+<a href="#lro-poll" name="lro-poll">:white_check_mark:</a> **DO** use the following pattern to allow clients to poll the current state of a Status Monitor resource:
 
-<a href="#lro-status-monitor-retry-after" name="lro-status-monitor-retry-after">:white_check_mark:</a> **DO** include a `Retry-After` header in the response to GET requests to the status monitor if the operation is not complete. The value of this header should be an integer number of seconds to wait before making the next request to the status monitor.
+```text
+GET /operations/<operation-id>?api-version=<api-version>
```

Comment:
nit, may better to make URL consistent with the PUT, to make it clear they are the same URL (just different HTTP method)

```suggestion
GET <operation-endpoint>/<operation-id>?api-version=<api-version>
```